config/default/kustomization.yaml

```diff
@@ -1,19 +1,14 @@
 # Adds namespace to all resources.
-namespace: cluster-api-provider-maas-system
+namespace: capm-system
```

Comment:
do you think we should use capm or capmaas ? I don't know personally, just asking.  
metal3 provider is using capm3.

Reply:
Ack